src/pages/HomePage/index.js

```diff
@@ -32,8 +47,16 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function HomePage() {
+const HomePage = (props) => {
+  const { home = [] } = props;
+
   const classes = useStyles();
+
+  useEffect(() => {
+    props.fetchHomeConfiguration();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
why do you need to disable eslint here?

Reply:
fixed. update will be in next commit

Reply:
How have you fixed it?
If you can't fix it, tell me a reason.